cluster/gce/trusty/configure.sh

```diff
@@ -144,8 +144,17 @@ install_additional_packages() {
   fi
 }
 
-# Downloads kubernetes binaries and salt tarball, unpacks them, and places them
-# to suitable directories.
+# Retry a download until we get it.
+#
+# $1 is the file to creat
+# $2 is the URL to download
+download_or_bust() {
```

Comment:
Ah, yes, the code mismatches the comment. Which do you think is more reasonable, loop permanently until successfully downloading it, or try several times then fail? Originally I simply copied the code form configure-vm.sh, but later I thought permanently loop may not make much sense.